openelevationservice/server/api/querybuilder.py

```diff
@@ -14,7 +14,7 @@
 log = get_logger(__name__)
 
 coord_precision = float(SETTINGS['coord_precision'])
-division_limit = 1 / float(SETTINGS['maximum_nodes'])
+# division_limit = 1 / float(SETTINGS['maximum_nodes'])
```

Comment:
Delete the commented out code if it is not necessary in the future

Reply:
DONE